lib/tasks/lint.js

```diff
@@ -0,0 +1,41 @@
+/*eslint-env node*/
+
+'use strict';
+
+var Builder = require('broccoli').Builder;
+var reject = require('rsvp').reject;
+var Funnel = require('broccoli-funnel');
+var eslint = require('broccoli-lint-eslint');
+
+function runLint() {
+  var ui = this.ui;
+  var hadError = false;
+
+  var tree = new Funnel(this.project.root, {
+    include: [
+      'app/**/*.js',
+      'tests/**/*.js'
+    ]
+  });
+  var eslintTree = eslint(tree, {
+    console: {
+      log: function(message) {
+        hadError = true;
```

Comment:
Hmm, I guess that would depend on the user's configuration for fail-on-warn.  What would be the best way to look that up?